src/Orchard.Web/Modules/Orchard.OpenId/Views/EditorTemplates/Parts.AzureActiveDirectorySettings.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@model Orchard.OpenId.Models.AzureActiveDirectorySettingsPart
+
+<h2>Azure Active Directory Settings</h2>
+<fieldset>
+    @Html.LabelFor(m => m.Tenant, T("Tenant"))
+    @Html.TextBoxFor(m => m.Tenant, new { @class = "text large" })
+    <span class="hint">@T("Azure Active Directory tenant (e.g. yoursite.onmicrosoft.com).")</span>
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.ADInstance, T("Active Directory Instance"))
+    @Html.TextBoxFor(m => m.ADInstance, new { @class = "text large" })
+    <span class="hint">@T("Default instance is https://login.microsoftonline.com/{your-tenant-name}")</span>
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.ClientId, T("App ID"))
+    @Html.TextBoxFor(m => m.ClientId, new { @class = "text large" })
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.AppName, T("App Name"))
+    @Html.TextBoxFor(m => m.AppName, new { @class = "text large" })
+    <span class="hint">@T("The application name you wish to give active directory login rights to.")</span>
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.LogoutRedirectUri, T("Logout Redirect"))
+    @Html.TextBoxFor(m => m.LogoutRedirectUri, new { @class = "text large" })
+    <span class="hint">@T("Redirect url after azure logout, default is http://localhost:30321/OrchardLocal/")</span>
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.ServiceResourceID, T("Service Resource ID"))
+    @Html.TextAreaFor(m => m.ServiceResourceID, new { @class = "text large" })
+    <span class="hint">
+        @T(@"If you have a single 'Service Resource ID' just write it down directly.
+             If you have multiple resources, enter each resource id on its own line, using key=value pairs.
+             Example: service1=https://yoursite.onmicrosoft.com/some-guid-for-service1")
+    </span>
+</fieldset>
+<fieldset>
+    @Html.LabelFor(m => m.AppKey, T("App Key"))
+    @Html.TextBoxFor(m => m.AppKey, new { @class = "text large" })
+</fieldset>
+<fieldset>
+    @Html.CheckBoxFor(m => m.BearerAuthEnabled)
+    @Html.LabelFor(m => m.BearerAuthEnabled, T("Enable Bearer Token Authentication").ToString(), new { @class = "forcheckbox" })
```

Comment:
Don't use `LabelFor` and `T.ToString` at the same time or there will be a double encoding. Use a `<label>` tag instead. Everywhere in this PR.